ERR_PNPM_INCLUDED_DEPS_CONFLICT

docker 部署
重新打包的镜像在拉取py插件和xiaoyao图鉴插件的时候都会出现这个错误
```
 ================ 
miao-yunzai    |  [信息]  更新 py-plugin 运行依赖  
miao-yunzai    |  ================ 
miao-yunzai    | 
miao-yunzai    |  ERR_PNPM_INCLUDED_DEPS_CONFLICT  modules directory (at "/app/Miao-Yunzai") was installed with optionalDependencies, dependencies. Current install wants optionalDependencies, dependencies, devDependencies.
```
尝试把docker/docker-entrypoint.sh 脚本文件中的pnpm install -P 修改为 pnpm install  未解决
在gitee上看到大佬回复
https://gitee.com/yoimiya-kokomi/Yunzai-Bot/issues/I6UYG7
疑似pnpm版本过高出现的问题

--- a/docker/Dockerfile
+++ b/docker/Dockerfile
@@ -49,7 +49,7 @@ RUN export BUNDLE_FFMPEG=${BUNDLE_FFMPEG:-true} \
     \
     && _NPM_MIRROR_FLAG="" \
     && if [ "$USE_NPM_MIRROR"x = "true"x ]; then _NPM_MIRROR_FLAG="--registry=https://registry.npmmirror.com"; fi \
-    && npm install pnpm -g $_NPM_MIRROR_FLAG \
+    && npm install pnpm@7.30.0 -g $_NPM_MIRROR_FLAG \
     \
     && ((test "$BUNDLE_POETRY"x = "true"x \
     && apt-get update \

